libs/community/langchain_community/vectorstores/azuresearch.py

```diff
@@ -625,7 +625,7 @@ def semantic_hybrid_search_with_score_and_rerank(
                     },
                 ),
                 float(result["@search.score"]),
-                float(result["@search.reranker_score"]),
+                float(result["@search.reranker_score"]) if result["@search.reranker_score"] else None,
```

Comment:
In what circumstances will the score be None?

Reply:
Do you have a test case that shows this behavior? I haven't seen it be None before.

Reply:
Hello @lawadam and  @ccurme , I’m currently involved in the Adaptive RAG agent process at Ernst & Young. Our first step is to classify the query. Following that, we utilize Azure Search in conjunction with a vector store for asimilarity_search of the “semantic_hybrid” type. However, we’re encountering an error in this specific instance. Due to the proprietary nature of my organization’s work, I’m unable to share the code or test case with you.